cis/system/vm-hsp-block-ubuntu-6.2.10.yaml

```diff
@@ -0,0 +1,17 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: vm-hsp-block-ubuntu-6.2.10
+spec:
+  tags: ["CIS", "rule-6.2.10", "Ubuntu", "rhosts"]
+  message: "Unauthorised modification or access to rhosts files detected ."
+  file:
+    severity: 1
+    matchPatterns:
+    - pattern: /**/**/.rhosts
+    - pattern: /**/.rhosts
```

Comment:
@rohitrishim please add `/.rhosts` also so that the file cannot be created under `/` as well